### DISCUSSION TITLE
Clarifying the rules on exploration task (Level 2)

### DISCUSSION THREAD

**Discussion by AhmedBaza1** [larics/icuas22_competition#53] [Q&A] [answered]
Hello, 
Should we map all area of  Zone 2 or just provid a map a long the way while navigating  from zone 1 to zone 3. It was unclear to me in the rule book as it was mentioned this way “mapping Zone 2 along the way” 
Could you please provite more explanation on this.

**Comment by fpetric** [comment, marked as answer]
Hi @AhmedBaza1, thanks for the question.

As we already discussed [here](https://github.com/larics/icuas22_competition/discussions/27#discussioncomment-2324528), you do not need to submit a map. We were looking into evaluating the quality of the map but finally decided against it. Therefore, you do not actually need to map anything in Zone 2 (however, you can map if it helps you navigate the obstacles).
